sihl_core/lib/run.ml

```diff
@@ -39,16 +42,20 @@ end
 module Project : PROJECT = struct
   open Opium.Std
 
-  type t = { apps : (module APP) list; config : Config.Setting.t }
+  type t = {
+    apps : (module APP) list;
+    config : Config.Setting.t;
```

Comment:
Now that I'm looking at this, Config.Settings.t looks a bit weird. Should it be Config.t or Setting.t instead?

Reply:
Agree, the namespaces aren't optimized for being intuitive yet :)